lesscss/utilities/helpers.less

```diff
@@ -10,3 +10,9 @@
   flex-shrink: 1;
   overflow-y: scroll;
 }
+
+.screen-reader-only {
+  position: absolute !important;
```

Comment:
Why is `!important` needed here?

Reply:
You have a rule like this in your style sheet:

```css
* {
  position: relative;
}
```

I'm using table elements in my PR and React gives off a warning saying that `position: relative` doesn't play nice with tables.

To get rid of the warning (and avoid potential styling headaches) I set my own CSS rule to counteract it:

```css
.combatants, .combatants * {
      /* position relative produces a warning in the console */
    position: static;
}
```

However `.combatants *` overwrites `.screen-reader-only` so I added `!important` to overpower `.combatants *`.

For `.screen-reader-only` it is probably better to add `!important` to all three lines than to try and keep `!important` out of the rule completly. If you are adding a `screen-reader-only` class to something, you want those styles to work reliably. you don't want other styles to overwrite them accidentally.

I'm going to add `!important` to the other 2 rules as well. There aren't any breakages around them at the moment but there could be in the future.